### DISCUSSION TITLE
Quick terminal like in iTerm2

### DISCUSSION THREAD

**Discussion by frolvanya** [ghostty-org/ghostty#3665] [Q&A] [answered]
Is it possible to make ghostty quick terminal behave like dedicated hotkey window in iTerm2?
1) I need to be able to open it in fullscreen mode, since I'm using drop-down (quick terminal) as a main terminal window
2) For some reason quick terminal works only if I'm focused on the main ghostty window. Is it possible to make it work when I call it anywhere from the system?

**Comment by ronit18** [comment, marked as answer]
you can set keybind using this `keybind = global:cmd+alt+s=toggle_quick_terminal` it will give you ghostty quick-terminal everywhere but not in fullscreen.

**Reply by frolvanya** [reply]
Thanks for the help, just found about `global:` modifier as well. Somehow missed it in the docs during the first inspection
